### ISSUE TITLE
Implement and integrate NER

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#46]
## User story
1. As a developer
2. I want / need to a module within the Q&A transformation pipeline to incorporate the NER processing
3. So that the NER-annotated training data is prepared for training

## Acceptance criteria
* Integrate the NER module and insure it aligns with our data transforming pipeline
* Implement the NER to enhance the training data
* Ensure the data is prepared for model training

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
